cmd/odf/set/log_level.go

```diff
@@ -0,0 +1,50 @@
+package set
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/red-hat-storage/odf-cli/pkg/rook/common"
+	"github.com/spf13/cobra"
+)
+
+var setLogLevelCmd = &cobra.Command{
+	Use:                "log-level",
+	Short:              "Set different log levels for ceph dameons like mon, osd and mds",
+	Example:            "odf set log-level osd 20",
+	DisableFlagParsing: true,
+	PreRunE:            validateLogLevelArgs,
+	Args:               cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		clientsets := root.GetClientsets(ctx)
+		common.SetLogLevel(ctx, clientsets, root.OperatorNamespace, root.StorageClusterNamespace, args)
+	},
+}
+
+func validateLogLevelArgs(_ *cobra.Command, args []string) error {
+	isValidSubsytem := false
+	for _, subsystem := range common.ValidCephSubSystems {
+		if subsystem == args[0] {
+			isValidSubsytem = true
+			break
+		}
+	}
+
+	if !isValidSubsytem {
+		return fmt.Errorf("invalid ceph subsystem %q. Supported subsystems : %+v", args[0], common.ValidCephSubSystems)
+	}
+
+	// validate log level
+	match, err := regexp.MatchString("^[1-9][0-9]?$", args[1])
```

Comment:
According to the [ceph logging](https://docs.ceph.com/en/latest/rados/troubleshooting/log-and-debug/#ceph-subsystems), we could also allow the separate log levels for the in-memory setting, so it could be two numbers separated by a slash such as "1/5". I don't expect it will commonly be needed, but seems better not to restrict it.